app/views/exportImport.html

```diff
@@ -72,33 +64,41 @@
             
           </div>
 
-          <div ng-if="importProgressStatus">
+          <div ng-if="importStatus">
             <h3>Import Results: </h3>
-            <progressbar ng-if="importProgressStatus" style="width: 100%; margin-bottom: 15px;" class="progress-striped active" max="100" value="importProgressStatus" type="{{importProgressType}}"><i>{{importProgressStatus}} %</i></progressbar>
-            <div>
-              <label>Successfull Imports ( {{importSuccess}} )</label>
-            </div>
-            <div>
-              <label class="failedImportsLink">Failed Imports ( {{importFail}} )</label>
 
-              <div id="no-more-tables" ng-if="importFail">
-                <table class="table table-bordered" style="100%">
+            <img src="images/loading.gif" alt="Loading..." ng-if="importStatus == 'progress'"/>
+            
+            <label class="btn btn-primary" ng-click="showConflictModal()" ng-if="importStatus == 'resolvingConflicts'"><strong>Resolve Conflicts</strong></label>
+
+            <div ng-if="importStatus == 'done'">
+              <div>
+                <label class="successfulImports">Successfull Imports ( {{ numberOfSuccessfulImports() }} )</label>
```

Comment:
I know this was inherited from before, but I don't think the look of change this to `Successful Imports ( 5 )` I think we can change this to `Successful Imports: 5` for this one and the others.